### PR TITLE
Explain how transaction aggregation works

### PR DESCRIPTION
**Pull request by ahey**
Transaction aggregation can be prevented through the use of unique custom remitter names.

### CODE REVIEW

**Review by antulik — APPROVED** [review]
LGTM

**Comment by antulik** [comment]
Also there was a recent change to disable debit aggregation. Not sure if we want to publish that https://app.asana.com/0/798383606298853/1141717314457372/f

File: source/openapi3/split.yaml [hunk 1]
```diff
@@ -947,6 +947,17 @@ info:
...
 
+    ## Aggregation
+
+
+    When Split detects multiple debits or credits to be processed against a 
+    single bank account, they are aggregated into one transaction.
+
+    If you assign a unique remitter name for each transaction, this prevents 
+    the transactions from being aggregated. This is useful if you would like
+    to ensure that your transactions are processed individually.
```
Comment (trev) [inline source/openapi3/split.yaml]: Consider restructuring to:

> Split will automatically aggregate debits that are:
>
> * From the same bank account; and
> * Initiated by the same Split account.
>
> Likewise for credits:
>
> * To the same bank account; and
> * Initiated by the same Split account.
>
> Should you wish for aggregation to be disabled, please contact support@splitpayments.com.au.
> Note that additional charges may apply.

**Review by trev — COMMENTED** [review]
Thanks @ahey - some suggestions.

**Pull request merged**